PractiseJuly7/Program.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PractiseJuly7
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //foreach (var item in EmployeeFilter.Filter(Employee.GetEmployees(), filter => filter.LastName == "Lannister"))
+            //{
+            //    Console.WriteLine($"{item.FirstName}, {item.BirthDate.ToShortDateString()}");
+            //}
+            //DeferedExample();
+            CollectionExample();
+        }
+
+        public static void CollectionExample()
+        {
+            Employee[] firstCollection = new Employee[] {
+                new Employee{ FirstName = "Tatiana", LastName = "Ryzhik", Age = 38, Gender = "F", Company = "ParkMe"},
+               new Employee{ FirstName = "Olga", LastName = "Pavlenko", Age = 36, Gender = "F", Company = "ParkMe"},
+               new Employee{ FirstName = "Helen", LastName = "Zub", Age = 20, Gender = "F", Company = "Max"},
+               new Employee{ FirstName = "Kirill", LastName = "Novikov", Age = 26, Gender = "M", Company = "ParkMe"},
+            };
+            Employee[] secondCollection = new Employee[] {
+                new Employee{ FirstName = "Helen", LastName = "Zub", Age = 20, Gender = "F", Company = "Max"},
+               new Employee{ FirstName = "Kirill", LastName = "Novikov", Age = 26, Gender = "M", Company = "ParkMe"},
+               new Employee{ FirstName = "Alex", LastName = "Matsegora", Age = 25, Gender = "M", Company = "Max"},
+               new Employee{ FirstName = "Oleg", LastName = "Reshetilo", Age = 26, Gender = "M", Company = "Admin"},
+               new Employee{ FirstName = "Valeriy", LastName = "Malinovskiy", Age = 22, Gender = "M", Company = "Here"},
+            };
+
+            var collectionWithDuplicates = firstCollection.Concat(secondCollection);
+            Console.WriteLine("With duplicates:");
+            Display(collectionWithDuplicates);
+
+            var collectionWithoutDuplicates = firstCollection.Union(secondCollection);
+            Console.WriteLine("Without duplicates:");
+            Display(collectionWithoutDuplicates);
+
+            var collectionIntersection = firstCollection.Intersect(secondCollection);
+            Console.WriteLine("Employees in both collections (intersection):");
+            Display(collectionIntersection);
+
+            List<Employee> firstList = firstCollection.ToList();
+            firstList.Add(new Employee { FirstName = "Jane", LastName = "Gladysheva", Age = 25, Gender = "F", Company = "Here" });
+            Console.WriteLine("Employees of first collection plus one more:");
+            Display(firstList);
+        }
+        public static void DeferedExample()
```

Comment:
Please do not forget to add empty lines between class members.

Reply:
fixed